ons_alpha/release_calendar/models.py

```diff
@@ -80,7 +80,11 @@ class ReleasePage(BasePage):
         related_name="+",
     )
 
-    is_accredited = models.BooleanField("Accredited Official Statistics", default=False)
+    is_accredited = models.BooleanField(
+        "Accredited Official Statistics",
+        default=False,
+        help_text="If ticked, will display an information block about the data being accredited official statistics and include the accredited logo",
```

Comment:
note: you could also use the `help_text` kwarg on `FieldPanel` - https://docs.wagtail.org/en/stable/reference/pages/panels.html#help-text.